### ISSUE TITLE
jQuery Standards Group?

### ISSUE THREAD

**Issue by aladdin-add** [jquery-foundation/standards#64]
it has been renamed to js foundation

**Comment by jorydotcom** [comment]
yes @aladdin-add that's correct - when the foundation transitioned from the jQuery Foundation to the JS Foundation so too did our standards engagement work. Can you point to where this is referenced so it can be corrected?

**Comment by aladdin-add** [comment]
seems some of the names in `readme.md` have not been updated. https://github.com/JSFoundation/standards#jquery-standards-group
